openmetadata-ui/src/main/resources/ui/playwright/e2e/Pages/DataInsightSettings.spec.ts

```diff
@@ -0,0 +1,220 @@
+import {
+  getApiContext,
+  redirectToHomePage,
+  toastNotification,
+} from '../../utils/common';
+import { settingClick } from '../../utils/sidebar';
+
+test.use({ storageState: 'playwright/.auth/admin.json' });
+
+test.describe.serial('Data Insight settings page should work properly', () => {
+  test.beforeEach(async ({ page }) => {
+    await redirectToHomePage(page);
+
+    const appsResponse = page.waitForResponse(
+      `/api/v1/apps?limit=15&include=non-deleted`
+    );
+
+    await settingClick(page, GlobalSettingOptions.APPLICATIONS);
+    await appsResponse;
+  });
+
+  test('Edit data insight application', async ({ page }) => {
+    const appResponse = page.waitForResponse(
+      `/api/v1/apps/name/DataInsightsApplication?fields=owners%2Cpipelines&include=all`
+    );
+
+    // Click on the config button
+    await page.click(
+      '[data-testid="data-insights-application-card"] [data-testid="config-btn"]'
+    );
+
+    await appResponse;
+
+    // Verify response status code (this is done implicitly by the route fulfillment)
+    // Click on the edit button
+    await page.getByTestId('edit-button').click();
+
+    // Select cron type
+    await page.getByTestId('cron-type').click();
+    await page.click('.rc-virtual-list [title="Day"]');
+
+    // Select hour
+    await page.click('[data-testid="hour-options"]');
+    await page.click('#hour-select_list + .rc-virtual-list [title="06"]');
+
+    // Select minute
+    await page.click('[data-testid="minute-options"]');
+    await page.click('#minute-select_list + .rc-virtual-list [title="00"]');
+
+    // Click on deploy button
+    await page.click('.ant-modal-body [data-testid="deploy-button"]');
+
+    await toastNotification(page, 'Schedule saved successfully');
+
+    // Verify response status code (this is done implicitly by the route fulfillment)
+    // Verify cron string
+    const cronString = await page
+      .locator('[data-testid="cron-string"]')
+      .textContent();
+
+    expect(cronString).toContain('At 06:00 AM');
+  });
+
+  test('Uninstall application', async ({ page }) => {
+    const appResponse = page.waitForResponse(
+      `/api/v1/apps/name/DataInsightsApplication?fields=owners%2Cpipelines&include=all`
+    );
+
+    // Click on the config button
+    await page.click(
+      '[data-testid="data-insights-application-card"] [data-testid="config-btn"]'
+    );
+
+    await appResponse;
+
+    // Verify response status code (this is done implicitly by the route fulfillment)
+    // Click on the manage button
+    await page.click('[data-testid="manage-button"]');
+
+    // Click on the uninstall button
+    await page.click('[data-testid="uninstall-button-title"]');
+
+    // Click on the save button
+    await page.click('[data-testid="save-button"]');
+
+    await toastNotification(page, 'Application uninstalled successfully');
+
+    await page
+      .locator('[data-testid="data-insights-application-card"]')
+      .waitFor({ state: 'hidden' });
```

Comment:
instead of checking state hidden, we should check for not exist here! it should be assertion!